airflow/www_rbac/views.py

```diff
@@ -1923,7 +1923,10 @@ def prefill_form(self, form, pk):
             d = {}
 
         for field in self.extra_fields:
-            value = d.get(field, '')
+            try:
+                value = d.get(field, '')
+            except Exception:
```

Comment:
so the json.loads sometimes will return non dictionary object which causes the exception?

Reply:
Yes.  I'm not sure if we should log something here, but once in that state it becomes difficult to edit the extra field from the UI

Reply:
The extra field is user generated, so must be treated with care.  The particular problem I'm seeing is when it is the string "null", and then clicking on the edit button fails.  I've just added a logging.warning, but it's not clear to me if the message is accurate since this code path is probably used with forms that do not have a 'conn_id' field....but an ambiguous warning message may be better than the current user experience.

Reply:
lgtm